app/src/main/res/values-en/strings.xml

```diff
@@ -794,7 +794,7 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <!-- "stop" as in "bus stop". But could be a tram stop, railway stop etc. too -->
     <string name="quest_busStopName_title2">"What’s the name of this stop?"</string>
     <!-- "stop" as in "bus stop". But could be a tram stop, railway stop etc. too -->
-    <string name="quest_busStopRef_title2">"What’s the reference number of this stop?"</string>
+    <string name="quest_busStopRef_title2">"What’s the number of this stop?"</string>
```

Comment:
Re: the postbox ref - maybe for consistency but also because maybe "reference numbers" on bus stops may not always be numbers revert this change too?

Reply:
It really should be bus stop number.  See for example:
https://www.octa.net/Bus/Rider-Tools-and-Apps/Bus-Stop-Numbers/

This was one that I actually encounter mapping and found very strangley worded.

Reply:
I played with "What's the bus stop number of this stop?" but that's repetitive.  Maybe "What's the bus stop number here?" could work with the UI?

Reply:
I would only work if the word "bus" is not mentioned, because "bus stop" (or "streetcar stop" etc) is already mentioned just below the question

Reply:
But well, the quest is only shown in the US I think, so you are the expert. If it is always a number, then this is fine.

Reply:
This was previously discussed in #2126. Some bus agencies call them “stop numbers”, others like [New York City MTA](https://bustime.mta.info/wiki/Help/FindingAStopCode#HStopCodeataBusStop) call them “stop codes”, and still others like [San Francisco Muni](https://www.sfmta.com/routes/22-fillmore) call them “stop IDs”.

Reply:
Anyhow, _virtually all_ bus agencies in the U.S. use bus stop names primarily, as described in https://github.com/streetcomplete/StreetComplete/issues/2126#issuecomment-773802224. These codes are only a shortcut for looking up real-time departure information on an SMS hotline or a website.

Reply:
Bus stop numbers can contain letters.  This is not a problem 😂 

![image](https://user-images.githubusercontent.com/3254090/168501479-168c9429-f0e7-4689-ac71-328823576cd0.png)

Reply:
I agree, although from a usability standpoint, “ID” would be less confusing. Otherwise, someone who doesn’t regularly ride the bus might mistakenly enter the bus _route_ number that’s more prominently signposted.

Reply:
It is more important that it is not confused with another number (e.g. bus route number) that that the most common wording is used. So in that manner, it would be OK if it sounds a little weird as long as there is no confusion.

However, not knowing the US-American system of numbered/coded bus stops, I do not feel confident to make a decision here. @ZeLonewolf or @1ec5, I leave the decision which wording to choose to you. Note that this exact question is also at least asked for streetcar stops.

Reply:
Is _this specific string_ also used for streetcar stops or is it a different one?

Reply:
> Is this specific string also used for streetcar stops or is it a different one?

Yes, that specific string is used. But below the question, the name of the feature (e.g. "bus stop") is shown

Reply:
> No because the user is prompted to enter a number, not answer yes/no

Does the prompt also give an option for "there is no stop number"?  I would think that would still make sense from a usability perspective, but here's an alternative that avoids the problem:

> What number or code identifies this stop?

Reply:
> Does the prompt also give an option for "there is no stop number"?

Yes, in "other answers..." -> "No reference number visible..."

> What number or code identifies this stop?

Ok. (though, the string for "No reference number visible..." should be changed then too)

Reply:
Updated both strings in c12b506.  Hopefully that hits the mark!  Note that the original version just said "No number visible" and omitted "reference".  I changed it to "No number or code visible"